src/main/target/KFC32F3_INAV/hardware_revision.c

```diff
@@ -0,0 +1,81 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+#include "platform.h"
+
+#include "drivers/system.h"
+#include "drivers/bus_spi.h"
+#include "drivers/sensor.h"
+#include "drivers/io.h"
+#include "drivers/gpio.h"
+#include "drivers/exti.h"
+
+#include "hardware_revision.h"
+
+uint8_t hardwareRevision = 1;
+
+void detectHardwareRevision(void)
```

Comment:
There is no early_setup() function in inav/BF code, but sometimes if you have board with a lot of SPI devices you need to force all CS signals to high, to be sure that sensors will be detected correctly. Without this sometimes board have a problem with initialization.